exercises/sum-of-multiples/canonical-data.json

```diff
@@ -38,6 +38,15 @@
       },
       "expected": 23
     },
+    {
+      "description": "multiples of 3 or 5 up to 20",
+      "property": "sum",
+      "input": {
+        "factors": [3, 5],
+        "limit": 10
```

Comment:
if the limit is to be 10, this case is the same as the above case which also has `[3, 5]` up to 10. please clarify how this case is different from the above case.